pixl_core/tests/test_project_config.py

```diff
@@ -30,16 +30,20 @@ def test_config_from_file():
 
     assert project_config.project.name == "test-extract-uclh-omop-cdm"
     assert project_config.project.modalities == ["DX", "CR"]
-    assert project_config.tag_operation_files == [
-        PROJECT_CONFIGS_DIR / "tag-operations" / "test-extract-uclh-omop-cdm.yaml"
-    ]
+    assert (
+        project_config.tag_operation_files.base
+        == PROJECT_CONFIGS_DIR / "tag-operations" / "test-extract-uclh-omop-cdm.yaml"
+    )
     assert project_config.destination.dicom == "ftps"
     assert project_config.destination.parquet == "ftps"
 
 
 BASE_YAML_DATA = {
```

Comment:
why aren't we just reading in the yaml file here? Feels like we'll always have to duplicate any changes

Reply:
Move schema loading logic to `project_config.py`
Have wrapper class around `PixlConfig` and tag operations loaded as dictionaries